Caboodle/Permissions/Permissions.android.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.Caboodle
 {
     internal static partial class Permissions
     {
-        static TaskCompletionSource<PermissionStatus> tcs;
+        static readonly object locker = new object();
+
+        static Dictionary<PermissionType, Tuple<int, TaskCompletionSource<PermissionStatus>>> requests =
```

Comment:
Should we be using the new Tuple Types?

Reply:
This requires a nuget package right? but just for build time stuff?